Cognite.Simulator.Utils/DefaultClasses/DefaultConnectorRuntime.cs

```diff
@@ -70,6 +70,7 @@ public static async Task RunStandalone()
                 catch (Exception e)
                 {
                     logger.LogError(e, "Unhandled exception: {message}", e.Message);
+                    throw;
```

Comment:
this was the bug, the exception was being eaten up

Reply:
let's do the same with the other place https://github.com/cognitedata/dotnet-simulator-utils/commit/ffbd236a18d849da5ee2762c34de3eb92d01ac12#diff-d3f097be5eb10746d715e992e40f702614c001eda5855955464f6e236a2fcfe1R137